src/main/java/org/commcare/session/CommCareSession.java

```diff
@@ -118,39 +118,21 @@ public CommCareSession(CommCareSession oldCommCareSession) {
         }
     }
 
-    public Vector<Entry> getEntriesForCommand(String commandId) {
-        return getEntriesForCommand(commandId, false);
-    }
-
-    public Vector<Entry> getEntriesForCommand(String commandId, boolean includeNested) {
-        return getEntriesForCommand(commandId, new OrderedHashtable<String, String>(), includeNested);
-    }
-
     /**
      * @param commandId          the current command id
-     * @param currentSessionData all of the datums already on the stack
-     * @param includeNested     whether to include entries from child modules
      * @return A list of all of the form entry actions that are possible with the given commandId
      * and the given list of already-collected datums
      */
-    private Vector<Entry> getEntriesForCommand(String commandId,
-                                               OrderedHashtable<String, String> currentSessionData,
-                                               boolean includeNested) {
+    private Vector<Entry> getEntriesForCommand(String commandId) {
         Vector<Entry> entries = new Vector<>();
         if (commandId == null) {
             return entries;
         }
         for (Suite s : platform.getInstalledSuites()) {
             List<Menu> menusWithId = s.getMenusWithId(commandId);
             if (menusWithId != null) {
-                // make a copy so that we can use this as a queue
-                Queue<Menu> menusToExamine = new LinkedList<>(menusWithId);
-                while (!menusToExamine.isEmpty()) {
-                    Menu menu = menusToExamine.poll();
-                    entries.addAll(getStillValidEntriesFromMenu(menu, currentSessionData));
-                    if (includeNested) {
-                        menusToExamine.addAll(s.getMenusWithRoot(menu.getId()));
-                    }
+                for (Menu menu : menusWithId) {
```

Comment:
Reverted back to this behavior and removed the logic surrounding `includeNested`